### ISSUE TITLE
TImm - resnet152

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#45]
**Source / repo**

https://github.com/huggingface/pytorch-image-models

**Model description**

[DESCRIPTION]

**Dataset**

imagenet
[URL]

**Literature benchmark source**

[URL]

**Literature benchmark performance**

[DESCRIPTION]
[VALUE/S]

**Strong Compute result achieved**

[VALUE/S]

**Basic training config (as applicable)**

Nodes: 12
Epochs: 300
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

**Logs gist**

[URL]
